docs/Remix.md

```diff
@@ -435,7 +476,7 @@ const getSupabaseUrlFromRequestUrl = (url: string) => {
 };
 ```
 
-**Tip**: Some of this code is really PostgREST-specific. The `prefer` header is required to let PostgREST return one record instead of an array containing one record in response to `getOne` requests. A proxy for another CRUD API will require different parameters.
+**Tip**: Some of this code is really PostgREST-specific. The `prefer` header is required to let PostgREST returns one record instead of an array containing one record in response to `getOne` requests. A proxy for another CRUD API will require different parameters.
```

Comment:
Nope, grammar was good before.

```suggestion
**Tip**: Some of this code is really PostgREST-specific. The `prefer` header is required to let PostgREST return one record instead of an array containing one record in response to `getOne` requests. A proxy for another CRUD API will require different parameters.
```